aws/src/test/java/org/apache/iceberg/aws/s3/TestVendedCredentialsProvider.java

```diff
@@ -48,7 +49,12 @@
 public class TestVendedCredentialsProvider {
 
   private static final int PORT = 3232;
-  private static final String URI = String.format("http://127.0.0.1:%d/v1/credentials", PORT);
+  private static final String CREDENTIALS_URI =
+      String.format("http://127.0.0.1:%d/v1/credentials", PORT);
```

Comment:
Technically you can point to an URI, but I think we need to include a test pointing to a path consistent with the spec (e.g. `/v1/{prefix}/namespaces/{namespace}/tables/{table}/credentials`) and we should test both absolute and relative (which I believe this change will address as well.)